lib/json_to_graphql/parser.rb

```diff
@@ -22,9 +22,19 @@ def build_nodes(hash, parent_node)
     end
 
     def print
-      root.children.each_with_object("{\n") do |child, query|
-        query.concat("#{child.print("", 1)}")
-      end.concat("}")
+      if input_hash.key?('query')
```

Comment:
I think this should include `query`, `mutation`, or `subscription`. All of these could be operation types.